api/src/services/__tests__/competenciesService.ts

```diff
@@ -0,0 +1,19 @@
+import { listCompetencies } from '../competenciesService';
+import { mockQuery } from '../mockDb';
+
+describe('listCompetencies', () => {
+  it('should load a page that lists the competencies in the database with their titles and descriptions.', async () => {
```

Comment:
Should this test description be re-written to describe what is actually being tested?

Reply:
yes, agree. New comment: should query for the lists of competencies in the database with their titles and descriptions